infra/src/main/resources/exception/exceptionInfo.json

```diff
@@ -202,5 +202,29 @@
     "id": 35,
     "reason": "Shard Operator received an invalid URL for the BridgeIngress endpoint.",
     "type": "PLATFORM"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.platform.UnspecifiedProvisioningFailureException",
+    "id": 36,
+    "reason": "An unspecified error occurred handling your request. Please contact the Help Desk for support quoting the date and time of this error.",
+    "type": "PLATFORM"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.platform.ProvisioningTimeOutException",
+    "id": 37,
+    "reason": "Your request was not complete within the configured time-out. Please contact the Help Desk for support quoting the date and time of this error.",
+    "type": "PLATFORM"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.platform.ProvisioningMaxRetriesExceededException",
+    "id": 38,
+    "reason": "Your request failed after the maximum number of configured re-try attempts. Please contact the Help Desk for support quoting the date and time of this error.",
+    "type": "PLATFORM"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.platform.ProvisioningReplicaFailureException",
+    "id": 39,
+    "reason": "A ReplicaSet failed to be provisioned for your request. Please contact the Help Desk for support quoting the date and time of this error.",
```

Comment:
I'm personally -1 on the "contact the Help Desk" part of these errors. I think the first part is all that the user needs.

Reply:
Removed.